app/experimenter/experiments/tasks.py

```diff
@@ -198,6 +198,10 @@ def update_status(experiment):
         if experiment.status == Experiment.STATUS_COMPLETE:
             bugzilla.update_bug_resolution(experiment)
 
+    if recipe_data and (is_paused(recipe_data) != experiment.paused):
+        experiment.paused = not experiment.paused
```

Comment:
Instead of flipping it here, why not just get whatever value is returned by the API and store it directly?  Why add the complication of making it dependent on the relationship between experimenters state and Normandy's state? What if they become inconsistent?

Reply:
did not think the case where they become inconsistent. will change to store whatever API gives